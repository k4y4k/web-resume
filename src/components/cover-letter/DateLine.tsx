import dayjs from "dayjs";
import localizedFormat from "dayjs/plugin/localizedFormat";
import React from "react";
dayjs.extend(localizedFormat);

const DateLine = () => {
  return (
    <p
      className="text-orchid-700 font-bold uppercase text-xl font-mono "
      style={{ fontSize: "21px" }}
    >
      {dayjs().format("LL")}
    </p>
  );
};

export default DateLine;
