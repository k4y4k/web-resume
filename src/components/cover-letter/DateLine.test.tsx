import * as React from "react";
import DateLine from "./DateLine";
import MockDate from "mockdate";
import { render } from "@testing-library/react";

beforeEach(() => {
  MockDate.set(new Date("2021-06-15"));
});

afterEach(() => {
  MockDate.reset();
});

describe("DateLine", () => {
  test("renders OK", () => {
    const { container } = render(<DateLine />);

    expect(container).toMatchSnapshot();
  });
});
