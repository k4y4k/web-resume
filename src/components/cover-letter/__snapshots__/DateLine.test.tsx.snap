// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateLine renders OK 1`] = `
.c0 {
  --tw-text-opacity: 1;
  color: rgba(77,54,140,var(--tw-text-opacity));
  font-weight: 700;
  text-transform: uppercase;
  font-size: 1.25rem;
  line-height: 1.75rem;
  font-family: Inconsolata,ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace;
}

<div>
  <p
    class="c0"
    style="font-size: 21px;"
  >
    June 15, 2021
  </p>
</div>
`;
